src/store/accounts/account.spec.tsx

```diff
@@ -34,7 +34,7 @@ describe('account state', () => {
     const { result, waitForNextUpdate } = renderHook(() => useAtomValue(walletConfigStore), {
       wrapper,
     });
-    await waitForNextUpdate();
+    await waitForNextUpdate({ timeout: 10000 }); // 10 secs?
```

Comment:
I was pretty certain this was 10 seconds until that comment with a question mark threw me

Reply:
yeah lol was more "is this enough time?" 1 second is the default, and it's too short for the CI for some reason